src/query/functions/temporal/base.go

```diff
@@ -78,12 +81,9 @@ func (o baseOp) Node(controller *transform.Controller, opts transform.Options) t
 	}
 }
 
-// baseNode is an execution node
+// baseNode is an execution node.
 type baseNode struct {
-	op baseOp
-	// controller uses an interface here so we can mock it out in tests.
-	// TODO: use an exported interface everywhere instead of *transform.Controller.
-	// https://github.com/m3db/m3/issues/1430
```

Comment:
Should we leave the note about #1430 or just close the issue if we don't intend to do this anymore?

Reply:
Ah yeah my bad; we did turn one of these helper structs into an interface but it wasn't `transform.Controller`